plugins/node/opentelemetry-instrumentation-knex/README.md

```diff
@@ -0,0 +1,69 @@
+# OpenTelemetry knex Instrumentation for Node.js
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`knex`](https://github.com/knex/node-knex) and allows the user to automatically collect trace data and export them to their backend of choice.
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-knex
+```
+### Supported Versions
+
+ - `0.x.x`
```

Comment:
Sorry, I think I was not clear with my previous comment.
IMO, the clearest text to write here is ">=0.10.0" as this is what the instrumentation will attempt to patch.
But that's just a personal style, current text is also good if you prefer it.

Reply:
Why >=0.10 as opposed to >=0.8?

Reply:
```
export const SUPPORTED_VERSIONS = [
  // use "lib/execution" as basepath, latest tested 0.95.6
  '>=0.22.0',
  // use "lib" as basepath
  '>=0.10.0 <0.18.0',
  '>=0.19.0 <=0.20.10',
  '>=0.20.11 <0.22.0',
  // use "src" as basepath
  '>=0.18.0 <0.19.0',
];
```

These are the versions that are given to `InstrumentationNodeModuleDefinition`
So for `knex` version 0.8 there will be no patching and for every version >=0.10.0, the instrumentation will patch the module.